### ISSUE TITLE
unittest.loader._FailedTest.unit.modules.test_libcloud_compute test failures

### ISSUE THREAD

**Issue by garethgreenaway** [saltstack/salt#53033]
2019.2 failed [salt-fedora-28-py3](https://jenkinsci.saltstack.com/job/2019.2/job/salt-fedora-28-py3/80/testReport/junit/unittest.loader._FailedTest.unit/modules/test_libcloud_compute), [salt-fedora-29-py3](https://jenkinsci.saltstack.com/job/2019.2/job/salt-fedora-29-py3/80/testReport/junit/unittest.loader._FailedTest.unit/modules/test_libcloud_compute)
---

Failed to import test module: unit.modules.test_libcloud_compute Traceback (most recent call last):   File "/usr/lib64/python3.7/unittest/loader.py", line 436, in _find_test_path     module = self._get_module_from_name(name)   File "/usr/lib64/python3.7/unittest/loader.py", line 377, in _get_module_from_name     __import__(name)   File "/tmp/kitchen/testing/tests/unit/modules/test_libcloud_compute.py", line 20, in <module>     from libcloud.compute.base import (   File "/usr/local/lib/python3.7/site-packages/libcloud/compute/base.py", line 33, in <module>     from libcloud.pricing import get_size_price   File "/usr/local/lib/python3.7/site-packages/libcloud/pricing.py", line 35, in <module>     from libcloud.utils.connection import get_response_object   File "/usr/local/lib/python3.7/site-packages/libcloud/utils/connection.py", line 17, in <module>     from libcloud.common.base import Connection   File "/usr/local/lib/python3.7/site-packages/libcloud/common/base.py", line 27, in <module>     import requests   File "/usr/lib/python3.7/site-packages/requests/__init__.py", line 43, in <module>     import urllib3   File "/usr/lib/python3.7/site-packages/urllib3/__init__.py", line 8, in <module>     from .connectionpool import (   File "/usr/lib/python3.7/site-packages/urllib3/connectionpool.py", line 11, in <module>     from .exceptions import (   File "/usr/lib/python3.7/site-packages/urllib3/exceptions.py", line 2, in <module>     from .packages.six.moves.http_client import ( ModuleNotFoundError: No module named 'urllib3.packages.six' 

```

ImportError: Failed to import test module: unit.modules.test_libcloud_compute
Traceback (most recent call last):
  File "/usr/lib64/python3.7/unittest/loader.py", line 436, in _find_test_path
    module = self._get_module_from_name(name)
  File "/usr/lib64/python3.7/unittest/loader.py", line 377, in _get_module_from_name
    __import__(name)
  File "/tmp/kitchen/testing/tests/unit/modules/test_libcloud_compute.py", line 20, in <module>
    from libcloud.compute.base import (
  File "/usr/local/lib/python3.7/site-packages/libcloud/compute/base.py", line 33, in <module>
    from libcloud.pricing import get_size_price
  File "/usr/local/lib/python3.7/site-packages/libcloud/pricing.py", line 35, in <module>
    from libcloud.utils.connection import get_response_object
  File "/usr/local/lib/python3.7/site-packages/libcloud/utils/connection.py", line 17, in <module>
    from libcloud.common.base import Connection
  File "/usr/local/lib/python3.7/site-packages/libcloud/common/base.py", line 27, in <module>
    import requests
  File "/usr/lib/python3.7/site-packages/requests/__init__.py", line 43, in <module>
    import urllib3
  File "/usr/lib/python3.7/site-packages/urllib3/__init__.py", line 8, in <module>
    from .connectionpool import (
  File "/usr/lib/python3.7/site-packages/urllib3/connectionpool.py", line 11, in <module>
    from .exceptions import (
  File "/usr/lib/python3.7/site-packages/urllib3/exceptions.py", line 2, in <module>
    from .packages.six.moves.http_client import (
ModuleNotFoundError: No module named 'urllib3.packages.six'

		
```

**Comment by KChandrashekhar** [comment]
Test is failing on 2019.2 branch and not on 2019.2.1 so it will be fixed in 2019.2.2 release

**Comment by DmitryKuzmenko** [comment]
The same is for these tests
unittest.loader._FailedTest.unit.modules.test_libcloud_loadbalancer
unittest.loader._FailedTest.unit.modules.test_libcloud_storage
unittest.loader._FailedTest.unit.runners.test_vault
unittest.loader._FailedTest.unit.states.test_grafana_datasource
